src/Structure/Content/MenuInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Defines the contract for registering a menu for WordPress.
+ *
+ * @author  Chrispian Burks <chrispian.burks@webdevstudios.com>
+ * @package WebDevStudios\OopsWP\Structure\Content
+ * @since 2019-05-01
+ */
+
+namespace WebDevStudios\OopsWP\Structure\Content;
+
+use WebDevStudios\OopsWP\Utility\Registerable;
+
+/**
+ * Interface MenuInterface
+ *
+ * @author  Chrispian Burks <chrispian.burks@webdevstudios.com>
+ * @package WebDevStudios\OopsWP\Structure\Content
+ * @since   2019-05-01
+ */
+interface MenuInterface extends Registerable {
```

Comment:
I might have missed this before, but this is great! I like this pattern of providing an interface for each structure type.

Reply:
I can't take any credit for the idea. I was just following what I was seeing and thought it was the right way lol. Glad it's a good thing.